CMakeLists.txt

```diff
@@ -145,6 +145,13 @@ if(TRITON_BUILD_PYTHON_MODULE)
     add_link_options(${Python3_LINK_OPTIONS})
   endif()
 
+  if (DEFINED TRITON_EXTERNAL_CODEGEN_BACKEND_DIRS)
+    foreach(BACKEND_DIR ${TRITON_EXTERNAL_CODEGEN_BACKEND_DIRS})
+      cmake_path(APPEND BACKEND_DIR "build" OUTPUT_VARIABLE BACKEND_DIR_BUILD_OUTPUT)
```

Comment:
Please fix this before committing: output dir should be inside the current cmake output dir rather than inside the plug-ins source location